HW2/P5/driver.py

```diff
@@ -16,9 +16,26 @@
 def randcolor():
     return np.random.uniform(0.0, 0.89, (3,)) + 0.1
 
+def cmp_zorder(a, b):
+        j = 0
+        k = 0
+        x = 0
+        for k in range(2):
+            y = a[1][k] ^ b[1][k]
+            if less_msb(x, y):
+                j = k
+                x = y
+        return a[1][j] - b[1][j]
+
+def less_msb(x, y):
+        return x < y and x < (x ^ y)
+
+
```

Comment:
Interesting. I went for the Hilbert curve ordering, because I couldn't figure out a good z-ordered implementation.